### ISSUE TITLE
所有可以设置的参数都在training script example里吗？

### ISSUE THREAD

**Issue by lyrmagical** [bert-nmt/bert-nmt#52]
python train.py $DATAPATH \
-a $ARCH --optimizer adam --lr 0.0005 -s $src -t $tgt --label-smoothing 0.1 \
--dropout 0.3 --max-tokens 4000 --min-lr '1e-09' --lr-scheduler inverse_sqrt --weight-decay 0.0001 \
--criterion label_smoothed_cross_entropy --max-update 150000 --warmup-updates 4000 --warmup-init-lr '1e-07' \
--adam-betas '(0.9,0.98)' --save-dir $SAVEDIR --share-all-embeddings $warmup \
--encoder-bert-dropout --encoder-bert-dropout-ratio $bedropout | tee -a $SAVEDIR/training.log
上面是所有的可以设置的参数吗？如果我想训练的时候取消bert参数的固定该怎么做？

**Comment by teslacool** [comment]
You can finetune the BERT part with `--finetune-bert` option.

**Comment by lyrmagical** [comment]
Thanks for your answer! I am new to fairseq, how can I know what parameters in total can be set?

**Comment by teslacool** [comment]
You can see more in [model.py](https://github.com/bert-nmt/bert-nmt/blob/fcb616d28091ac23c9c16f30e6870fe90b8576d6/fairseq/models/transformer.py#L57) and [translation.py](https://github.com/bert-nmt/bert-nmt/blob/fcb616d28091ac23c9c16f30e6870fe90b8576d6/fairseq/tasks/translation.py#L114).

**Comment by lyrmagical** [comment]
Thanks a lot!
